src/store/helium/heliumStatusSlice.ts

```diff
@@ -1,67 +1,85 @@
 import { createAsyncThunk, createSlice } from '@reduxjs/toolkit'
-import { showMessage } from 'react-native-flash-message'
-import { formatDistanceToNow } from 'date-fns'
-import { getIncidents, getStatus } from '../../utils/StatusClient'
-import { Incident, Status, StatusInfo } from './StatusTypes'
-import shortLocale from '../../utils/formatDistance'
+import AsyncStorage from '@react-native-community/async-storage'
+import { getIncidents } from '../../utils/StatusClient'
+import { Incident, StatusInfo } from './StatusTypes'
 
-export type HeliumStatusState = {
-  status: {
-    page?: StatusInfo
-    status?: Status
-  }
-  incidents: {
-    page?: StatusInfo
-    incidents?: Incident[]
-  }
-}
-const initialState: HeliumStatusState = {
-  status: {},
-  incidents: {},
+const STATUS_KEY = 'statusKey'
+const ONBOARDING_COMPONENT_ID = '74lkk1qwp5xq'
+const APP_COMPONENT_ID = 'qfcq2xt6v0xm'
+const COMPONENT_IDS = [ONBOARDING_COMPONENT_ID, APP_COMPONENT_ID]
```

Comment:
will we want this to be configurable via wallet api? or are we ok with it being hardcoded and releasing a new app version to update these in the future?

Reply:
Good question. I don't imagine this will change often but will check with Coco.